### PR TITLE
Compilation of a variety of proofreading and similar QOL changes to emanual

### PR DESCRIPTION
**Pull request by Jonathon-RINC**

### CODE REVIEW

**Comment by Jonathon-RINC** [comment]
@robotpilot 
I believe that all conflicts should be resolved and this compilation of changes should be eligible for merge.

Please let me know if anything else is required.

File: _includes/en/dxl/x/baudrate.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<<<<<<< HEAD
```
Comment (robotpilot) [inline _includes/en/dxl/x/baudrate.md:1]: There seems to be a conflict here, please review it again.

File: _includes/en/dxl/x/indirect_data.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+**NOTE**: Only addresses in the [RAM area (Address 64 ~ 227)](#control-table-of-ram-area) of the control table may be assigned to Indirect Addresses.
+{: .notice}
+
+| Indirect Address Range | Description                                                                           |
+|:-----------------------:|:--------------------------------------------------------------------------------------|
+| 64 ~ 661               | [EEPROM](#control-table-of-eeprom-area) addresses can't be assigned an Indirect Address |
+
+#### [Indirect Address and Indirect Data Examples](#indirect-address-and-indirect-data-examples)
+
+<<<<<<< HEAD
```
Comment (robotpilot) [inline _includes/en/dxl/x/indirect_data.md:43]: Please double-check this - if you see the "<<<<<" or ">>>>>" mark, you should look carefully.

File: _includes/en/dxl/y/output_bearing_y_m.md [hunk 3]
```diff
@@ -8,35 +8,53 @@
...
 
 ## [Allowable dynamic equivalent load, Pc_max](#allowable-dynamic-equivalent-load-pc_max)
 
 ![](/assets/images/dxl/y/bearing_formula/m_2.png)   
 
-- L<sub>10</sub> - Bearing life [hour] @ 20,000 hour
+- L<sub>10</sub> - Bearing service life [hour] @ 20,000 hour
 - N<sub>n</sub> - Nominal output speed [rpm]
 - C - Basic dynamic load rating [N]
+<<<<<<< HEAD
```
Comment (robotpilot) [inline _includes/en/dxl/y/output_bearing_y_m.md:38]: Please double-check this and this page.

**Review by robotpilot — CHANGES_REQUESTED** [review]
I've found issues in a few places and left comments on each code. 
Please take note and reconsider.

**Comment by Jonathon-RINC** [comment]
Thank you for the review, I believe that all the reported issues have been resolved in the latest commits just added.

**Review by robotpilot — APPROVED** [review]
LGTM, Thanks @Jonathon-RINC :)

**Comment by robotpilot** [comment]
HI, @ROBOTIS-Yang 
@Jonathon-RINC had a request to improve the manual. 
Overall, it's very good. I'd like to ask you to give it a final review and merge it if there are no issues, 
but if you find any issue, please let me know. 
Have a nice day. :)

File: _includes/en/dxl/assembly/xl430_frame_assembly.md
Comment (ROBOTIS-Yang) [inline _includes/en/dxl/assembly/xl430_frame_assembly.md:1]: Please add the links

**Comment by robotpilot** [comment]
@Jonathon-RINC, 
We've found some things to fix, so we'll change the branch to merge and do the merge first, 
and then we'll push a fix commit to that branch so we can do the final merge. Thank you.

**Comment by Jonathon-RINC** [comment]
@robotpilot 
Understood, thank you for working with me to get these changes merged.

**Pull request merged**